app/src/main/java/com/example/habitsmasher/UserDatabaseHelper.java

```diff
@@ -0,0 +1,80 @@
+package com.example.habitsmasher;
+
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.firestore.DocumentReference;
+import com.google.firebase.firestore.DocumentSnapshot;
+import com.google.firebase.firestore.FirebaseFirestore;
+
+import java.util.List;
+
+public class UserDatabaseHelper {
```

Comment:
Just a helper class that stores methods that fetch the user's follower/following counts and setting them on the front end